00_organization/README.md

```diff
@@ -0,0 +1,15 @@
+# Organization and Introduction to Research Software Engineering
+
+Learning goals:
+
+- Lecture, lab organization, and the challenge
+- What is research software engineering and what is special about it?
+- Why do we need research software engineering?
+
+| Duration | Content |
+| --- | --- |
+| 20 minutes | [`challenge_intro.md`](https://github.com/Simulation-Software-Engineering/Lecture-Material/blob/main/00_organization/challenge_intro_slides.md) |
+| 40 minutes | [`intro_course.md`](https://github.com/Simulation-Software-Engineering/Lecture-Material/blob/main/00_organization//intro_course_slides.md) |
+| 10 minutes | [`organizational_remarks_week3_slides.md`](https://github.com/Simulation-Software-Engineering/Lecture-Material/blob/main/00_organization/organizational_remarks_week3_slides.md) |
+| 5 minutes | [`organizational_remarks_week7_slides.md`](https://github.com/Simulation-Software-Engineering/Lecture-Material/blob/main/00_organization/organizational_remarks_week7_slides.md) |
+| 15 minutes | [`rse_basics.md`](https://github.com/Simulation-Software-Engineering/Lecture-Material/blob/main/organization/material/rse_basics_slides.md) |
```

Comment:
Should this chapter ("RSE Basics") be in the organization chapter or somewhere else? Is it organization or rather actual content?

Reply:
@uekerman suggestions?

Reply:
I would keep it here. Not worth a separate chapter right now. Maybe next year when we have this "all-in-one intro example", we could do a "Introduction" chapter.